ADAL/src/request/ADAuthenticationRequest+WebRequest.m

```diff
@@ -117,7 +119,7 @@ - (NSString*)generateQueryStringForRequestType:(NSString*)requestType
                                  OAUTH2_REDIRECT_URI, [[_requestParams redirectUri] adUrlFormEncode],
                                  OAUTH2_STATE, state];
     
-    [startUrl appendFormat:@"&%@", [[ADLogger adalId] adURLFormEncode]];
+    [startUrl appendFormat:@"&%@", [[ADLogger adalMetadata] adURLFormEncode]];
```

Comment:
we don't send app name/version to auth endpoint? :)

Reply:
Yeah, I followed the logic from before to minimize changes...